src/test/resources/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImplFulfilmentTest.CachedCase.json

```diff
@@ -11,5 +11,6 @@
   "addressType": "HH",
   "caseType": "HH",
   "estabType": "Household",
-  "region": "E"
+  "region": "E",
+  "ceOrgName": "The Cached Case Organisation"
```

Comment:
please add new line to end of JSON file (remove github red circle warning) for consistency

Reply:
done